easypermissions/src/main/java/pub/devrel/easypermissions/AppSettingsDialog.java

```diff
@@ -105,9 +105,9 @@ private void startForResult(Intent intent) {
     /**
      * Display the built dialog.
      */
+    @SuppressWarnings("NewApi")
     public void show() {
         if (mNegativeListener == null) {
-            //noinspection NewApi The Builder constructor prevents this
```

Comment:
@samtstern There's a bug in lint so I had to suppress it for the whole method. 😕